rotkehlchen/chain/manager.py

```diff
@@ -1736,6 +1736,9 @@ def edit_eth2_validator(self, validator_index: int, ownership_proportion: FVal)
             validator_index=validator_index,
             ownership_proportion=ownership_proportion,
         )
+        self.flush_cache('query_ethereum_beaconchain_balances')
```

Comment:
Would have been better to have moved those before the db.edit_validator() function just as is already done below in the add_validator